internal/services/newrelic/new_relic_monitor_resource_test.go

```diff
@@ -247,3 +293,35 @@ resource "azurerm_role_assignment" "test" {
 }
 `, template, data.RandomInteger, data.Locations.Primary, effectiveDate, email)
 }
+
+func (r NewRelicMonitorResource) monitoredSubscription(data acceptance.TestData, effectiveDate string, email string) string {
+	template := r.template(data)
```

Comment:
Minor, but we don't need a variable for this, we can just pass this as an argument to the string formatter 
```suggestion
```

Reply:
fixed